### DISCUSSION TITLE
语音文件哪里找？还是需要自己上传？

### DISCUSSION THREAD

**Discussion by GeraintGui** [fishaudio/fish-speech#359] [Q&A]
Error: Invalid value for '--input-path' / '-i': Path 'paimon.wav' does not exist.
推理部分的每次跑每次都说没有paimon.wav，这个文件是要自己找吗？

**Comment by feuyeux** [comment]
随便找个文件替换即可 比如川建国的声音：
https://fish.audio/text-to-speech/?modelId=5196af35f6ff4a0dbf541793fc9f2157&taskId=5fed90320b0a4bb293b7e6415a16f3e8

然后执行：
python fish_speech/models/vqgan/inference.py \
    -i "DonaldTrump.mp3" \
    --checkpoint-path "checkpoints/fish-speech-1.5/firefly-gan-vq-fsq-8x1024-21hz-generator.pth"
